cvat/apps/engine/background.py

```diff
@@ -320,15 +330,13 @@ def _handle_rq_job_v1(
             # Such dependencies are never removed or finished,
             # as there is no TTL for deferred jobs,
             # so the current job can be blocked indefinitely.
+            cancel_and_reenqueue(rq_job)
+
+        if is_result_outdated():
+            cancel_and_reenqueue(rq_job)
+
+        return Response(status=status.HTTP_202_ACCEPTED)
```

Comment:
Probably, all 202 should have the same output format. Currently, there are cases with and without data in the body.